pkg/broker/broker_test.go

```diff
@@ -64,3 +65,176 @@ func ExampleBrokerService_IsEnabled() {
 	// Output: true
 	// false
 }
+
+func ExampleBrokerService_ServiceDefinition() {
+	service := BrokerService{
+		Name: "left-handed-smoke-sifter",
+		DefaultServiceDefinition: `{"id":"abcd-efgh-ijkl"}`,
+	}
+
+	// Default definition
+	defn, err := service.ServiceDefinition()
+	fmt.Printf("%q %v\n", defn.ID, err)
+
+	// Override
+	viper.Set(service.DefinitionProperty(), `{"id":"override-id"}`)
+	defn, err = service.ServiceDefinition()
+	fmt.Printf("%q %v\n", defn.ID, err)
+
+	// Bad Value
+	viper.Set(service.DefinitionProperty(), "nil")
+	_, err = service.ServiceDefinition()
+	fmt.Printf("%v\n", err == nil)
```

Comment:
It could make sense to print the error so that the target audience gets an idea of what it might look like. Could be helpful for them  to be better able to identify this kind of error later on while debugging.

Reply:
Prefixed with something like, "Example error: %v\n"